src/Adapter/ElFinder/PhpcrDriver.php

```diff
@@ -54,7 +53,7 @@ class PhpcrDriver extends ElFinderVolumeDriver
     /**
      * @var array
      */
-    private $options;
+    protected $options;
```

Comment:
why change the visibility of this?

Reply:
I'm getting this otherwise:

`PHP Fatal error:  Access level to Symfony\Cmf\Bundle\MediaBundle\Adapter\ElFinder\PhpcrDriver::$options must be protected (as in class elFinderVolumeDriver) or weaker in .../vendor/symfony-cmf/media-bundle/src/Adapter/ElFinder/PhpcrDriver.php on line 34`

Reply:
hm, so that means we collide on that field. does the new volume driver also have a constructor that we should be calling? how does it use the options field? we probably need to rename this field as we would overwrite the options of the base class...

Reply:
Yes its weird. Thing is, with the previous ElFinderVolumeDriver (https://github.com/helios-ag/ElFinderPHP/blob/master/src/Driver/ElFinderVolumeDriver.php) it was done in the same way.

They talk about this issue here too https://github.com/symfony/symfony/issues/25353.

Reply:
we merge the options with the options of the parent class. hm, whatever. i guess its fine then.